tests/conftest.py

```diff
@@ -167,9 +179,9 @@ def selenium(selenium: WebDriver) -> WebDriver:
 
 def create_user(request_session: requests.Session, server: str, auth: Tuple[str, str]):
     # check if user already exists before creating
-    r = request_session.get(f"{server}/", auth=auth)
+    r = request_session.get(server, auth=auth)
     if "user" not in r.json():
-        create_account_url = f"{server}/accounts/{auth[0]}"
+        create_account_url = urljoin(server, f"/accounts/{auth[0]}")
```

Comment:
I'm not sure this would work as expected:

```
>>> from urllib.parse import urljoin
>>> urljoin("http://server/v1", "/buckets")
'http://server/buckets'
```

Reply:
Good eye. This fixes it https://github.com/mozilla-services/kinto-dist/pull/1970/commits/ff29e25e902fc385330815b9b1b581a63dbaf6a1#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R189